disconnected/updating/index.adoc

```diff
@@ -1,42 +1,36 @@
 = About cluster updates in a disconnected environment
 include::_attributes/common-attributes.adoc[]
-:context: about-restricted-network-updates
+:context: about-disconnected-updates
 
 toc::[]
 
-////
-WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to this assembly for earlier versions should be done in separate PRs based off of their respective version branches. Otherwise, your cherry picks may fail.
-
-To do: Remove this comment once 4.13 docs are EOL.
-////
-
 A disconnected environment is one in which your cluster nodes cannot access the internet.
 For this reason, you must populate a registry with the installation images.
 If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap.
 If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
 
 A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
 
-[id="about-disconnected-updates-mirroring"]
+[id="about-disconnected-updates-mirroring_{context}"]
 == Mirroring {product-title} images
 To update your cluster in a disconnected environment, your cluster environment must have access to a mirror registry that has the necessary images and resources for your targeted update. The following page has instructions for mirroring images onto a repository in your disconnected cluster:
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring {product-title} images]
+* xref:../../disconnected/updating/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring {product-title} images]
 
-[id="about-disconnected-updates-update"]
+[id="about-disconnected-updates-update_{context}"]
 == Performing a cluster update in a disconnected environment
 
 You can use one of the following procedures to update a disconnected {product-title} cluster:
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#updating-restricted-network-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service]
+* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service]
 
-* xref:../../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update.adoc#updating-restricted-network-cluster[Updating a cluster in a disconnected environment without the OpenShift Update Service]
+* xref:../../disconnected/updating/disconnected-update.adoc#updating-disconnected-cluster[Updating a cluster in a disconnected environment without the OpenShift Update Service]
```

Comment:
[OTA-821][1] is coming up on two years old with my pitch for why OSUS-assisted updates are worth the setup cost.  Worth including any of those points as we reshuffle?  Or continue to punt on that, and leave the trade-offs as something customers have to work out on their own?

[1]: https://issues.redhat.com/browse/OTA-821

Reply:
So one of the things we hope to achieve with this overall reorg is, when we create the index page for this entire "disconnected environments" section, we want to include a subsection on that page that provides a sort of recommended/ideal path for the "typical user's" disconnected experience.

Basically it'll be a section where we recommend users use oc-mirror, the Agent-based Installer, and update using a local OSUS, and to not deviate unless user needs make it necessary (for example choosing a different installation method due to some constraint). 

I think it would be good to wait until that happens to see if that looks like enough "user steering", and if not then I'd be open to elaborating the OSUS vs no-OSUS advantages further somewhere in this OTA section